community/kernel/src/test/java/org/neo4j/kernel/impl/proc/ReflectiveProcedureWithArgumentsTest.java

```diff
@@ -156,6 +182,34 @@ public MyOutputRecord( String name )
         }
     }
 
+    public static class ClassWithProcedureWithDefaults
+    {
+        @Procedure
+        public Stream<MyOutputRecord> defaultValues( @Name( value = "a", defaultValue = "a") String a , @Name( value = "b", defaultValue = "42") long b, @Name( value = "c",
+                defaultValue = "3.14") double c)
```

Comment:
It is the unfortunate behaviour of annotations in java, you can only provide `compile time constants`, so you cannot do `defaultValue = Collections.emptyList()`, or even `defaultValue = null`